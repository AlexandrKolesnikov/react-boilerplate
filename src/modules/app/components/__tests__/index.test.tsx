import React from 'react';
import { shallow } from 'enzyme';
import App from '../index';

describe('App Module/Components', () => {
  it('<App /> should be rendered without children', () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });

  it('<App /> should be rendered with children', () => {
    const component = shallow(<App>Test</App>);

    expect(component).toMatchSnapshot();
  });
});
