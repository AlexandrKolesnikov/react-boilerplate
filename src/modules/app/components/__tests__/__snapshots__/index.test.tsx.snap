// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Module/Components <App /> should be rendered 1`] = `
<div
  className="app"
>
  <div
    className="app__main-container"
  >
    <main
      className="app__content"
    />
  </div>
</div>
`;

exports[`App Module/Components <App /> should be rendered with children 1`] = `
<div
  className="app"
>
  <div
    className="app__main-container"
  >
    <main
      className="app__content"
    >
      Test
    </main>
  </div>
</div>
`;
